src/Manager.java

```diff
@@ -0,0 +1,81 @@
+
+public class Manager {
+	
+	public static void main(String args[])
+	{
+		int turns = 1;
+		
+		boolean isCaught = false;
+		Hare hare = new Hare(0,0);
+		Hound hound = new Hound(0,0);
+		hound.setDirection(hare.getDirection());
+		while(!isCaught && turns <= 100)
+		{
+			System.out.println("Turn:" +turns);
+			if(turns <=3)
+			{
+				hare.move(3, hare.getDirection());
+			}
+			else 
+			{
+				if(Math.sqrt(Math.pow(getXDisplacement(hare,hound), 2) + Math.pow(getYDisplacement(hare,hound), 2)) <=2)
```

Comment:
First: Don't do the distance calculation inside the if statement - it should be in a separate method. Second: you're not actually comparing the distance to a given target value, like you are supposed to. You just call sqrt() and that's it.